printer_mods/JosAr/Klicky-Probe/README.md

```diff
@@ -0,0 +1,118 @@
+
+There are three gantry extrusion mounts possible:
+- one fixed to be used on the Voron V2.4 or V1.8 AB with MGN12 or MGN9
+<img src="Photos/Fixed_mount_complete.jpg" width="100">
+- one that has some variance for other toolheads
+<img src="Photos/Variable_mount_complete.jpg" width="100">
+- one fixed sidemount dock to allow a purge/scrub bucket on the left side of the bed
+<img src="Photos/Fixed_sidemount_complete.jpg" width="100">
+
+The fixed gantry extrusion mounts have been confirmed to work on the Voron V2.4 and V1.8
+
+The normal magnet installation is that the two magnets that attach to the microswitch are installed with the same polarity, the 3rd magnet should have the inverse polarity.
+There is however the possibility that the magnets will demagnetize over time due to the alternating magnetic fields thay may result in a slow but sure demagnetization of the magnets, the magnets are so strong that may take a long time to show the effects of demagnetization YMMV.
+
+It is not necessary any soldering the probe microswitch connectors are also press-fit on the magnets.
+<p float="left">
+  <img src="Photos/probe_components.jpg" width="150" />
+  <img src="Photos/probe_install.jpg" width="150" />
+  <img src="Photos/Probe_topside.jpg" width="150" />
+  <img src="Photos/probe_v1_underside.jpg" width="150" />
+  <img src="Photos/probe_complete.jpg" width="150" />
+</p>
+
+The AB mount wires are also connected with pressure from the magnets, you can use the probe magnets as a template to insert the AB mount magnets, it is easier that way to don't insert the magnets the wrong way.
+<p float="left">
+  <img src="Photos/AB_Mount_wiring_1.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_2.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_3.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_4.jpg" width="150" />
+  <img src="Photos/AB_Mount_wiring_complete.jpg" width="150" /> 
+</p>
+
+You will not lose Y travel on any configuration in the tests that were done.
+
+It is also recommended to glue the magnets in place, superglue is good.
+
+You will need to add macros to klipper to be able to dock and undock the probe as necessary to do the Endstop (if necessary) and Quad Gantry Level, it is in the Klipper Macro directory.
+
+<img src="Photos/All_Klicky_probe_components.jpg" width="600" />
+
+Probe BOM:
+- 1x microswitch (the omron D2F-5 or D2F-5L (removing the lever) is recommended)
+- 2x M2x10 self tapping
+- 4x 6x3 magnets
+
+AB mount BOM:
+- 3x 6x3 magnets
+
+Probe Dock:
+- 1x 6x3 magnets
+- 2x M3x20
+
+Fixed Dock mount:
+- 2x M3 threaded insert M3x5x4
+- 2x M5x10
+- 2x M5 t-nut or equivalent
+
+or 
+
+variable Dock mount:
+- 10x M3 threaded insert M3x5x4
+- 8x M3x8
+- 2x M5x10
+- 2x M5 t-nut or equivalent
+
+If you would like to check a possibly more uptodate repository, check ![here](https://github.com/jlas1/Klicky-Probe)
```

Comment:
Remove exclamation mark (which makes this an image), the link should be just

    [here](https://github.com/jlas1/Klicky-Probe)